client/src/modules/multiple_payroll_indice/modals/config.modal.html

```diff
@@ -21,16 +21,51 @@
         <div
         ng-show ="(rubConfigured.indice_to_grap && rubConfigured.is_indice)"
         ng-repeat ="rubConfigured in ConfigPaiementModalCtrl.rubrics | orderBy:'label'" class="form-group">
+        <div style="background-color: #EFEFEF;" ng-if="rubConfigured.is_linked_to_grade">
+          <div class="form-group">
+            <label class="control-label"> {{ConfigPaiementModalCtrl.rubValueFrequency}} : {{rubConfigured.label}} </label>
+            <input
+              type="number"
+              min="0"
+              class="form-control"
+              ng-model="ConfigPaiementModalCtrl.frequencyRubrics[rubConfigured.id]"
+              ng-keyup="ConfigPaiementModalCtrl.onUpdateRubricValue(rubConfigured.id)"
+              ng-required="true"
+              ng-key>
+          </div>
 
+          <bh-input-text
```

Comment:
Why are there three separate inputs here?  What do they correspond to? 

I see the text of https://github.com/IMA-WorldHealth/bhima/pull/7745#issuecomment-2391665924 that there are three separate inputs for bonus by grade.  Would it be possible to explain what they do and what impact they have to the user?  Maybe in a help message under the input?

I stared at it for a while, but I couldn't understand the difference between "bonus_by_grade", "frequency: bonus_by_grade" and "value: bonus_by_grade".

Reply:
> Why are there three separate inputs here? What do they correspond to?
> 
> I see the text of [#7745 (comment)](https://github.com/IMA-WorldHealth/bhima/pull/7745#issuecomment-2391665924) that there are three separate inputs for bonus by grade. Would it be possible to explain what they do and what impact they have to the user? Maybe in a help message under the input?
> 
> I stared at it for a while, but I couldn't understand the difference between "bonus_by_grade", "frequency: bonus_by_grade" and "value: bonus_by_grade".

The explanation is simple: for an item such as "**Heure supplémentaire**" that is configured to be linked to a grade, once the grades have been set for each item, during payroll configuration, the system will check the employee's grade for items that are grade-linked. The "**Heure supplémentaire**" field will indicate the configured value for this item and for the corresponding grade. The "Frequency" field will allow the input of the number of overtime hours, and the system will multiply the frequency by the value to calculate the total amount to be assigned to the **Heure supplémentaire**.

![image](https://github.com/user-attachments/assets/8e325c12-b2d3-4c36-98af-13b6b9c9dd99)

![image](https://github.com/user-attachments/assets/2b7ce374-c59b-4363-aa0c-9942cc0bcc14)

![image](https://github.com/user-attachments/assets/7051b571-fd93-4c37-aa93-a7b3ed27c311)

Reply:
![image](https://github.com/user-attachments/assets/8ffc889a-1567-4180-922b-c234f4a39dd9)
In this example, the employee has worked a total of **10 Heure supplémentaire**. The system will multiply these 10 hours by the configured value for their grade, which is 1.05 for overtime. Thus, 10 x 1.05 equals 10.5, and this value will be automatically assigned to the "Heure supplémentaire " rubric. Before this feature was introduced, payroll managers had to perform this calculation manually for each employee.